docs/development.rst

```diff
@@ -4,14 +4,61 @@
 Developing in pySMT
 ===================
 
-Building Documentation
-======================
+Licensing
+=========
 
-pySMT uses Sphinx for documentation. To build the documentation you
-will need sphinx installed, this can be done via pip.
+pySMT is distributed under the APACHE License (see LICENSE file). By
+submitting a contribution, you automatically accept the conditions
+described in LICENSE. Additionally, we ask you to certify that you have
+the right to submit such contributions. We adopt the "Developer 
+Certificate of Origin" approach as done by the Linux kernel.
 
-A Makefile in the docs/ directory allows to build the documentation in
-many formats. Among them, we usually consider html and latex.
+```
```

Comment:
The triple back-tick is Markdown syntax for verbatim. The Sphinx documentation uses restructured text (rst).
This needs to be adapted. See, e.g, [1].

This applies to a few other places, I'll try to mark them.

[1] http://openalea.gforge.inria.fr/doc/openalea/doc/_build/html/source/sphinx/rest_syntax.html#inserting-code-and-literal-blocks